commonui/src/main/res/values/strings.xml

```diff
@@ -35,5 +35,5 @@
     <string name="cancelButtonLabel">Cancel</string>
     <string name="edit" translatable="false">Edit</string>
     <string name="remove">Remove</string>
-    <string name="mobileAreYouSure">Are you sure?</string>
+    <string name="mobileAreYouSure">Remove this transfer method?</string>
```

Comment:
IMO best to rename the string name as well to match the content

Reply:
Resolved